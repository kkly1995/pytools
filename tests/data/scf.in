&CONTROL
   calculation     = 'scf'
   outdir          = 'pwscf_output'
   prefix          = 'pwscf'
   pseudo_dir      = './'
   tprnfor         = .true.
   wf_collect      = .true.
/

&SYSTEM
   degauss         = 0.0063338
   ecutwfc         = 300
   ibrav           = 0
   nat             = 54
   nosym           = .true.
   nspin           = 1
   ntyp            = 1
   occupations     = 'smearing'
   smearing        = 'fd'
   tot_charge      = 0
/

&ELECTRONS
   conv_thr        = 1e-10
/


ATOMIC_SPECIES 
   H  1.00794 H.qmc.soft.upf

ATOMIC_POSITIONS bohr
   H       -0.07327746       0.51826004       0.86149671 
   H        2.36615000       0.59153750       0.69043875 
   H        4.73230000       0.59153750       0.69043875 
   H        0.00000000       2.95768750       0.69043875 
   H        2.36615000       2.95768750       0.69043875 
   H        4.73230000       2.95768750       0.69043875 
   H        0.00000000       5.32383750       0.69043875 
   H        2.36615000       5.32383750       0.69043875 
   H        4.73230000       5.32383750       0.69043875 
   H       -1.18307500      -0.59153750       3.45219375 
   H        1.18307500      -0.59153750       3.45219375 
   H        3.54922500      -0.59153750       3.45219375 
   H       -1.18307500       1.77461250       3.45219375 
   H        1.18307500       1.77461250       3.45219375 
   H        3.54922500       1.77461250       3.45219375 
   H       -1.18307500       4.14076250       3.45219375 
   H        1.18307500       4.14076250       3.45219375 
   H        3.54922500       4.14076250       3.45219375 
   H       -2.36615000      -1.77461250       6.21394875 
   H       -0.00000000      -1.77461250       6.21394875 
   H        2.36615000      -1.77461250       6.21394875 
   H       -2.36615000       0.59153750       6.21394875 
   H       -0.00000000       0.59153750       6.21394875 
   H        2.36615000       0.59153750       6.21394875 
   H       -2.36615000       2.95768750       6.21394875 
   H       -0.00000000       2.95768750       6.21394875 
   H        2.36615000       2.95768750       6.21394875 
   H        1.18307500       0.59153750       2.07131625 
   H        3.54922500       0.59153750       2.07131625 
   H        5.91537500       0.59153750       2.07131625 
   H        1.18307500       2.95768750       2.07131625 
   H        3.54922500       2.95768750       2.07131625 
   H        5.91537500       2.95768750       2.07131625 
   H        1.18307500       5.32383750       2.07131625 
   H        3.54922500       5.32383750       2.07131625 
   H        5.91537500       5.32383750       2.07131625 
   H       -0.00000000      -0.59153750       4.83307125 
   H        2.36615000      -0.59153750       4.83307125 
   H        4.73230000      -0.59153750       4.83307125 
   H       -0.00000000       1.77461250       4.83307125 
   H        2.36615000       1.77461250       4.83307125 
   H        4.73230000       1.77461250       4.83307125 
   H       -0.00000000       4.14076250       4.83307125 
   H        2.36615000       4.14076250       4.83307125 
   H        4.73230000       4.14076250       4.83307125 
   H       -1.18307500      -1.77461250       7.59482625 
   H        1.18307500      -1.77461250       7.59482625 
   H        3.54922500      -1.77461250       7.59482625 
   H       -1.18307500       0.59153750       7.59482625 
   H        1.18307500       0.59153750       7.59482625 
   H        3.54922500       0.59153750       7.59482625 
   H       -1.18307500       2.95768750       7.59482625 
   H        1.18307500       2.95768750       7.59482625 
   H        3.54922500       2.95768750       7.59482625 

K_POINTS automatic
   6 6 6  1 1 1 

CELL_PARAMETERS bohr
         7.09845000       0.00000000       0.00000000 
         0.00000000       7.09845000       0.00000000 
        -3.54922500      -3.54922500       8.28526500 


