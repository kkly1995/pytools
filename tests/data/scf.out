
     Program PWSCF v.6.4 starts on 18May2020 at 13: 9:48 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      16 processor cores
     Number of MPI processes:                16
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     K-points division:     npool     =      16
     Reading input from scf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        4825    4825   1313               293091   293091   42055
 


     bravais-lattice index     =            0
     lattice parameter (alat)  =       7.0984  a.u.
     unit-cell volume          =     417.4779 (a.u.)^3
     number of atoms/cell      =           54
     number of atomic types    =            1
     number of electrons       =        54.00
     number of Kohn-Sham states=           32
     kinetic-energy cutoff     =     300.0000  Ry
     charge density cutoff     =    1200.0000  Ry
     convergence threshold     =      1.0E-10
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = SLA  PW   PBE  PBE ( 1  4  3  4 0 0)

     celldm(1)=   7.098450  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (  -0.500000  -0.500000   1.167194 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.428378 )  
               b(2) = (  0.000000  1.000000  0.428378 )  
               b(3) = (  0.000000  0.000000  0.856756 )  


     PseudoPot. # 1 for H  read from file:
     ./H.qmc.soft.upf
     MD5 check sum: a0038b7674ff7f611440715583e13524
     Pseudo is Norm-conserving, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 1076 points,  0 beta functions with: 

     atomic species   valence    mass     pseudopotential
        H              1.00     1.00794     H ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           H   tau(   1) = (  -0.0103230   0.0730103   0.1213641  )
         2           H   tau(   2) = (   0.3333333   0.0833333   0.0972661  )
         3           H   tau(   3) = (   0.6666667   0.0833333   0.0972661  )
         4           H   tau(   4) = (   0.0000000   0.4166667   0.0972661  )
         5           H   tau(   5) = (   0.3333333   0.4166667   0.0972661  )
         6           H   tau(   6) = (   0.6666667   0.4166667   0.0972661  )
         7           H   tau(   7) = (   0.0000000   0.7500000   0.0972661  )
         8           H   tau(   8) = (   0.3333333   0.7500000   0.0972661  )
         9           H   tau(   9) = (   0.6666667   0.7500000   0.0972661  )
        10           H   tau(  10) = (  -0.1666667  -0.0833333   0.4863306  )
        11           H   tau(  11) = (   0.1666667  -0.0833333   0.4863306  )
        12           H   tau(  12) = (   0.5000000  -0.0833333   0.4863306  )
        13           H   tau(  13) = (  -0.1666667   0.2500000   0.4863306  )
        14           H   tau(  14) = (   0.1666667   0.2500000   0.4863306  )
        15           H   tau(  15) = (   0.5000000   0.2500000   0.4863306  )
        16           H   tau(  16) = (  -0.1666667   0.5833333   0.4863306  )
        17           H   tau(  17) = (   0.1666667   0.5833333   0.4863306  )
        18           H   tau(  18) = (   0.5000000   0.5833333   0.4863306  )
        19           H   tau(  19) = (  -0.3333333  -0.2500000   0.8753952  )
        20           H   tau(  20) = (   0.0000000  -0.2500000   0.8753952  )
        21           H   tau(  21) = (   0.3333333  -0.2500000   0.8753952  )
        22           H   tau(  22) = (  -0.3333333   0.0833333   0.8753952  )
        23           H   tau(  23) = (   0.0000000   0.0833333   0.8753952  )
        24           H   tau(  24) = (   0.3333333   0.0833333   0.8753952  )
        25           H   tau(  25) = (  -0.3333333   0.4166667   0.8753952  )
        26           H   tau(  26) = (   0.0000000   0.4166667   0.8753952  )
        27           H   tau(  27) = (   0.3333333   0.4166667   0.8753952  )
        28           H   tau(  28) = (   0.1666667   0.0833333   0.2917984  )
        29           H   tau(  29) = (   0.5000000   0.0833333   0.2917984  )
        30           H   tau(  30) = (   0.8333333   0.0833333   0.2917984  )
        31           H   tau(  31) = (   0.1666667   0.4166667   0.2917984  )
        32           H   tau(  32) = (   0.5000000   0.4166667   0.2917984  )
        33           H   tau(  33) = (   0.8333333   0.4166667   0.2917984  )
        34           H   tau(  34) = (   0.1666667   0.7500000   0.2917984  )
        35           H   tau(  35) = (   0.5000000   0.7500000   0.2917984  )
        36           H   tau(  36) = (   0.8333333   0.7500000   0.2917984  )
        37           H   tau(  37) = (   0.0000000  -0.0833333   0.6808629  )
        38           H   tau(  38) = (   0.3333333  -0.0833333   0.6808629  )
        39           H   tau(  39) = (   0.6666667  -0.0833333   0.6808629  )
        40           H   tau(  40) = (   0.0000000   0.2500000   0.6808629  )
        41           H   tau(  41) = (   0.3333333   0.2500000   0.6808629  )
        42           H   tau(  42) = (   0.6666667   0.2500000   0.6808629  )
        43           H   tau(  43) = (   0.0000000   0.5833333   0.6808629  )
        44           H   tau(  44) = (   0.3333333   0.5833333   0.6808629  )
        45           H   tau(  45) = (   0.6666667   0.5833333   0.6808629  )
        46           H   tau(  46) = (  -0.1666667  -0.2500000   1.0699274  )
        47           H   tau(  47) = (   0.1666667  -0.2500000   1.0699274  )
        48           H   tau(  48) = (   0.5000000  -0.2500000   1.0699274  )
        49           H   tau(  49) = (  -0.1666667   0.0833333   1.0699274  )
        50           H   tau(  50) = (   0.1666667   0.0833333   1.0699274  )
        51           H   tau(  51) = (   0.5000000   0.0833333   1.0699274  )
        52           H   tau(  52) = (  -0.1666667   0.4166667   1.0699274  )
        53           H   tau(  53) = (   0.1666667   0.4166667   1.0699274  )
        54           H   tau(  54) = (   0.5000000   0.4166667   1.0699274  )

     number of k points=   108  Fermi-Dirac smearing, width (Ry)=  0.0063

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:   293091 G-vectors     FFT dimensions: (  80,  80, 108)

     Estimated max dynamical RAM per process >     456.27 MB

     Estimated total dynamical RAM >       7.13 GB

     Initial potential from superposition of free atoms

     starting charge   53.99954, renormalised to   54.00000
     Starting wfcs are   54 randomized atomic wfcs

     total cpu time spent up to now is       14.5 secs

     Self-consistent Calculation

     iteration #  1     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.4

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  5.75E-04,  avg # of iterations =  1.9

     total cpu time spent up to now is       69.5 secs

     total energy              =     -53.29617439 Ry
     Harris-Foulkes estimate   =     -53.28669779 Ry
     estimated scf accuracy    <       0.30947375 Ry

     iteration #  2     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.73E-04,  avg # of iterations =  1.1

     total cpu time spent up to now is       90.8 secs

     total energy              =     -53.29979362 Ry
     Harris-Foulkes estimate   =     -53.29710434 Ry
     estimated scf accuracy    <       0.02888786 Ry

     iteration #  3     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.35E-05,  avg # of iterations =  3.0

     total cpu time spent up to now is      115.5 secs

     total energy              =     -53.29974192 Ry
     Harris-Foulkes estimate   =     -53.30091124 Ry
     estimated scf accuracy    <       0.00230766 Ry

     iteration #  4     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.27E-06,  avg # of iterations =  3.2

     total cpu time spent up to now is      143.1 secs

     total energy              =     -53.30027352 Ry
     Harris-Foulkes estimate   =     -53.30030645 Ry
     estimated scf accuracy    <       0.00006956 Ry

     iteration #  5     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.29E-07,  avg # of iterations =  2.2

     total cpu time spent up to now is      168.8 secs

     total energy              =     -53.30028956 Ry
     Harris-Foulkes estimate   =     -53.30029161 Ry
     estimated scf accuracy    <       0.00000389 Ry

     iteration #  6     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.21E-09,  avg # of iterations =  2.2

     total cpu time spent up to now is      201.4 secs

     total energy              =     -53.30029041 Ry
     Harris-Foulkes estimate   =     -53.30029042 Ry
     estimated scf accuracy    <       0.00000004 Ry

     iteration #  7     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  7.67E-11,  avg # of iterations =  2.4

     total cpu time spent up to now is      228.9 secs

     total energy              =     -53.30029042 Ry
     Harris-Foulkes estimate   =     -53.30029042 Ry
     estimated scf accuracy    <          9.3E-09 Ry

     iteration #  8     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.72E-11,  avg # of iterations =  2.8

     total cpu time spent up to now is      255.4 secs

     total energy              =     -53.30029042 Ry
     Harris-Foulkes estimate   =     -53.30029042 Ry
     estimated scf accuracy    <          4.3E-09 Ry

     iteration #  9     ecut=   300.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  8.00E-12,  avg # of iterations =  2.2

     total cpu time spent up to now is      280.7 secs

     End of self-consistent calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     the Fermi energy is    15.7422 ev

!    total energy              =     -53.30029042 Ry
     Harris-Foulkes estimate   =     -53.30029042 Ry
     estimated scf accuracy    <          6.6E-11 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =      68.86282988 Ry
     hartree contribution      =       1.47195910 Ry
     xc contribution           =     -47.81737810 Ry
     ewald contribution        =     -75.81406019 Ry
     smearing contrib. (-TS)   =      -0.00364111 Ry

     convergence has been achieved in   9 iterations

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.01947689    0.01136662   -0.05419288
     atom    2 type  1   force =    -0.00179813    0.00049527   -0.00216316
     atom    3 type  1   force =    -0.00823163    0.00026504   -0.00682163
     atom    4 type  1   force =     0.00024484   -0.00836590   -0.00511662
     atom    5 type  1   force =     0.00055890   -0.00059857   -0.00011484
     atom    6 type  1   force =    -0.00050075   -0.00063722   -0.00042292
     atom    7 type  1   force =     0.00039527   -0.00082266   -0.00357401
     atom    8 type  1   force =     0.00070554    0.00048053    0.00054140
     atom    9 type  1   force =    -0.00066811    0.00049991   -0.00000388
     atom   10 type  1   force =    -0.00104790   -0.00526641    0.00514743
     atom   11 type  1   force =     0.00075525   -0.00212106    0.00157681
     atom   12 type  1   force =    -0.00002595   -0.00007940    0.00036276
     atom   13 type  1   force =    -0.00046979    0.00494873    0.00451038
     atom   14 type  1   force =     0.00033287    0.00181295    0.00060054
     atom   15 type  1   force =    -0.00021410   -0.00017499    0.00035203
     atom   16 type  1   force =    -0.00011039    0.00003687    0.00019986
     atom   17 type  1   force =    -0.00053801    0.00003677    0.00028913
     atom   18 type  1   force =     0.00057997    0.00035946   -0.00148497
     atom   19 type  1   force =    -0.00367411   -0.00055862    0.00298902
     atom   20 type  1   force =    -0.00006562    0.00010028    0.00030230
     atom   21 type  1   force =     0.00321578   -0.00025090    0.00290873
     atom   22 type  1   force =     0.00001715    0.00012151    0.00014319
     atom   23 type  1   force =     0.00054023    0.00055650   -0.00104423
     atom   24 type  1   force =    -0.00046621   -0.00036478    0.00070115
     atom   25 type  1   force =    -0.00196388    0.00018789    0.00119989
     atom   26 type  1   force =    -0.00019181   -0.00056164    0.00078025
     atom   27 type  1   force =     0.00187823    0.00034624    0.00080599
     atom   28 type  1   force =     0.01328723    0.00285636    0.00275912
     atom   29 type  1   force =    -0.00158212    0.00050760   -0.00248893
     atom   30 type  1   force =    -0.01797165    0.00431007    0.01822783
     atom   31 type  1   force =     0.00095556   -0.00002111    0.00051560
     atom   32 type  1   force =    -0.00153641    0.00131114   -0.00059720
     atom   33 type  1   force =     0.00000812    0.00067525    0.00110763
     atom   34 type  1   force =     0.00178466   -0.00331271    0.00216849
     atom   35 type  1   force =    -0.00126653   -0.00151524    0.00014004
     atom   36 type  1   force =    -0.00134107   -0.00504590    0.00374119
     atom   37 type  1   force =     0.00011900    0.00007944    0.00019277
     atom   38 type  1   force =     0.00050522   -0.00033740    0.00123035
     atom   39 type  1   force =    -0.00032273   -0.00046404    0.00152326
     atom   40 type  1   force =    -0.00014950   -0.00085664    0.00055529
     atom   41 type  1   force =     0.00045132    0.00022111    0.00089514
     atom   42 type  1   force =    -0.00007310    0.00046629    0.00165301
     atom   43 type  1   force =     0.00056034    0.00033452   -0.00110068
     atom   44 type  1   force =    -0.00103855    0.00017227    0.00112048
     atom   45 type  1   force =     0.00018948    0.00007646    0.00044705
     atom   46 type  1   force =    -0.00323922   -0.00074201    0.00326461
     atom   47 type  1   force =     0.00071992   -0.00033533    0.00119328
     atom   48 type  1   force =     0.00187486   -0.00895180    0.01058074
     atom   49 type  1   force =    -0.00138976   -0.00111563    0.00055995
     atom   50 type  1   force =     0.00117208   -0.00134859   -0.00077823
     atom   51 type  1   force =     0.00045249   -0.00126884   -0.00229274
     atom   52 type  1   force =    -0.00270103    0.00131938    0.00175621
     atom   53 type  1   force =     0.00005334    0.00119383    0.00127360
     atom   54 type  1   force =     0.00174353    0.00997915    0.00388042

     Total force =     0.071958     Total SCF correction =     0.000011

     Writing output data file pwscf.save/
 
     init_run     :      9.53s CPU      9.72s WALL (       1 calls)
     electrons    :    237.74s CPU    266.27s WALL (       1 calls)
     forces       :      1.22s CPU      1.25s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      8.65s CPU      8.77s WALL (       1 calls)
     potinit      :      0.46s CPU      0.50s WALL (       1 calls)
     hinit0       :      0.31s CPU      0.33s WALL (       1 calls)

     Called by electrons:
     c_bands      :    190.74s CPU    215.02s WALL (      10 calls)
     sum_band     :     41.61s CPU     45.26s WALL (      10 calls)
     v_of_rho     :      3.12s CPU      3.29s WALL (      10 calls)
     mix_rho      :      0.61s CPU      0.67s WALL (      10 calls)

     Called by c_bands:
     cegterg      :    121.60s CPU    131.76s WALL (      70 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :    114.04s CPU    115.96s WALL (     225 calls)
     g_psi        :      1.07s CPU      1.08s WALL (     148 calls)
     cdiaghg      :      0.37s CPU      8.34s WALL (     211 calls)

     Called by h_psi:
     h_psi:pot    :    113.37s CPU    115.10s WALL (     225 calls)
     vloc_psi     :    113.37s CPU    115.10s WALL (     225 calls)

     General routines
     fft          :      0.82s CPU      0.83s WALL (     102 calls)
     ffts         :      0.08s CPU      0.08s WALL (      10 calls)
     fftw         :    113.69s CPU    115.30s WALL (   13850 calls)
 
     Parallel routines
 
     PWSCF        :   4m12.70s CPU   4m42.65s WALL

 
   This run was terminated on:  13:14:30  18May2020            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
